### PR TITLE
Update bitlocker-management-for-enterprises.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Updated several URLs. 
Updated code blocks with syntax for PowerShell.

### CODE REVIEW

**Comment by illfated** [comment]
Just out of curiosity, is it possible that the 3 links containing the locale `/en-us/` can be modified to not use the en-us locale?
From earlier pull requests, I have noticed that it is usually preferred to make the URLs locale agnostic if possible.

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Please also take into the consideration the URL locale nuances as @illfated suggests. Thanks.

**Comment by baardhermansen** [comment]
> Just out of curiosity, is it possible that the 3 links containing the locale `/en-us/` can be modified not to use the en-us locale?
> From earlier pull requests, I have noticed that it is usually preferred to make the URLs locale agnostic if possible.

Yeah, that's just a cockup on my part.

**Comment by baardhermansen** [comment]
@illfated @e0i I seem to have managed to correct the locale in latest commit to this branch.

File: windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md [hunk 1]
```diff
@@ -134,10 +130,9 @@ PS C:\> Enable-BitLocker -MountPoint "C:" -EncryptionMethod XtsAes256 -UsedSpace
 
 [Shielded VMs and Guarded Fabric](https://blogs.technet.microsoft.com/windowsserver/2016/05/10/a-closer-look-at-shielded-vms-in-windows-server-2016/) 
 
-<br />
 
 **Powershell**
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]:
```suggestion
**PowerShell**
```

Comment (Dansimp) [reply windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]: Fixed instance of **PowerShell**

**Pull request merged**